synapse/storage/databases/main/schema/delta/56/unique_user_filter_index.py

```diff
@@ -46,7 +48,4 @@ def run_create(cur, database_engine, *args, **kwargs):
         select_clause,
     )
 
-    if isinstance(database_engine, PostgresEngine):
-        cur.execute(sql)
-    else:
-        cur.executescript(sql)
+    execute_statements_from_stream(cur, StringIO(sql))
```

Comment:
Why this change? It seems potentially inefficient?

Reply:
This is a delta so it doesn't really matter if its inefficient. We do this because we don't expose an `executescript` on `LoggingTranasction`, and implementing it would be a bit of a faff.